_pages/volunteer.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: splash
+classes: flowing
+title: "Get involved with pyOpenSci"
+author_profile: false
+published: true
+site-map: true
+permalink: /volunteer.html
+volunteer-mission:
+  - excerpt: "The vibrant and diverse pyOpenSci community is driven by volunteer Pythonistas that care deeply about the scientific Python open source software that drives open science."
+help-us:
+  - image_path:
+    title: "Lend a Hand on GitHub"
+    alt:
+    excerpt: "Got some time to help? Check out our GitHub Project Board for a list of current issues that we could use help with. Any issue that is tagged `help-wanted` in our repos is also fair game for anyone to tackle! We add anyone who contributes to pyOpenSci to our [community page](/our-community/). "
+    url: https://github.com/orgs/pyOpenSci/projects/3/views/1
+    btn_label: "> Check out our GitHub Help Wanted Board"
+    btn_class: btn--inverse
+  - image_path:
+    title: "Sign up to be a scientific Python package reviewer"
+    alt:
+    excerpt: "Finding reviewers is one of the more challenging parts of running a peer review process. We are always looking for new reviewers from a broad range of scientific domains. Some reviewers have extensive packaging expertise and others have domain expertise. We think that mix is great, so sign up today! If you are new to reviewing we are happy to support you through our peer review mentorship program."
+    url: https://forms.gle/GHfxvmS47nQFDcBM6
+    btn_label: "> Sign up now."
+    btn_class: btn--inverse
+  - image_path:
+    title: "Get involved as software peer review Editor"
+    alt:
+    excerpt: "We also often recruit new editors to support our peer review process. Keep an eye out on our [Discourse forum](https://pyopensci.discourse.group/) for calls for new editors. In the meantime if you are interested in learning more about the editor role, check out our [peer review guidebook](https://www.pyopensci.org/software-peer-review/). "
+    url: https://www.pyopensci.org/software-peer-review/how-to/editors-guide.html
+    btn_label: "> Click here to learn more about the editor role."
+    btn_class: btn--inverse
+---
+
+{% include feature_row id="volunteer-mission" type="center" %}
+
+<div class="pyos-section purple">
```

Comment:
The splash layout is super wide. so i created these custom divs that allow us to create a more "flowing" page layout. the purple creates a purple background. 

it could be cleaner in terms of css but this works for now!